icu4j/main/classes/core/src/com/ibm/icu/number/Precision.java

```diff
@@ -365,6 +366,13 @@ public Precision withMode(MathContext mathContext) {
     // PACKAGE-PRIVATE APIS //
     //////////////////////////
 
+    /**
+     * @internal
+     * @deprecated ICU internal only.
+     */
+    @Deprecated
+    public static final BogusRounder BOGUS_PRECISION = new BogusRounder();
```

Comment:
Made this public to give access to UsagePrefsHandler, for this line:

        if (micros.rounder == Precision.BOGUS_PRECISION) {

It is also references in NumberFormatterImpl, but for that, "protected" still works, perhaps because they're in the same package (`com.ibm.icu.number`, whereas UsagePrefsHandler is in `com.ibm.icu.impl.number`).